src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentResponse.cs

```diff
@@ -99,6 +101,7 @@ public class PaymentResponseObject
         public string PayerReference { get; }
         public string InitiatingSystemUserAgent { get; }
         public PricesListResponse Prices { get; }
+        public Guid? RecurrenceToken { get; }
```

Comment:
Shouldn't this be a `string`?
https://developer.swedbankpay.com/payment-instruments/card/other-features#recur It's defined as a string here.
Any opinions @asbjornu ?

Reply:
Agreed, the format of tokens shouldn't be restricted or interpreted in any way. They are just opaque identifiers.

Reply:
Ok. I understand that from your perspective you want to keep the options open.
I set it to Guid to make it more predictable on the client side, but I have now changed it to string (of unlimited length).